llvm/lib/Target/X86/Disassembler/X86Disassembler.cpp

```diff
@@ -1134,6 +1134,25 @@ static int getInstructionIDWithAttrMask(uint16_t *instructionID,
   return 0;
 }
 
+static bool isNF(InternalInstruction *insn) {
+  // NF bit is the MSB of aaa.
```

Comment:
This comment should be moved to line 1191. It's used to explain why we use `else if` for `aaaFromEVEX4of4`